tests/acceptance/WooCommerceDynamicSegmentsCest.php

```diff
@@ -58,6 +62,10 @@ public function _before(\AcceptanceTester $i) {
       ->withName(self::NUMBER_OF_ORDERS_SEGMENT)
       ->withWooCommerceNumberOfOrdersFilter()
       ->create();
+    $this->totalSpentSegment = $segmentFactory
```

Comment:
Why did you decide to only test if the created segment works? Would it be a better test if we created the segment using the UI?

Reply:
I knew about this test only. I will add a test for creation too.

Reply:
Thanks. Also, to save some time during the test execution I'd recommend doing everything in a single test and not adding another test. 
Check Rodrigo's segment he did a similar test.